docs/PacketFence_MSPKI_Quick_Install_Guide.asciidoc

```diff
@@ -0,0 +1,381 @@
+
+The following is an example on how to configure an EAP-TLS connection for Windows/Android/OS X/iOS
+
+image::docs/images/scep-ms-pki-eaptls-example.png[scaledwidth="100%",alt="Provisioner EAP-TLS configuration"]
+
+OS X/iOS require signing the provisioning profile with a Certification Authority already trusted by the device such as e.g. VeriSign.
+Configuring this has to be done in the 'Signing' tab in the "Apple devices". 
+
+image::docs/images/packetfence-pki-eaptls-sign-example.png[scaledwidth="100%",alt="Signing provisioner"]
+
+Fill out the fields with the contents of the Base64 encoded certificates.
+To extract this information from a pem formatted certificate, copy the file content included between the begin and end tag, not including the delimiters themselves.
+
+Certificate file example:
+
+----
+----- BEGIN CERTIFICATE -----
+1234567890asdfghjkl
+zxcvbnmqwertyuiop78
+----- END CERTIFICATE -----
+----
+
+Copy everything between the BEGIN and END lines, but not the lines themselves.
+Repeat this operation for the certificate key and intermediate certificate if any.
+
+Private key file example:
+
+----
+----- BEGIN PRIVATE KEY -----
+1234567890asdfghjkl
+zxcvbnmqwertyuiop78
+----- END PRIVATE KEY -----
+----
+
+
+Portal Profiles Configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Provisioners have to be enabled on the Portal Profiles configuration in the PacketFence GUI.
+
+Under Configuration > Main > Portal Profiles, select each of the provisioners created above which should be active for the profile.
+If no portal profile is defined, configure the "default" profile to use the provisioners created.
+
+NOTE: If you use two different portal profiles for the open and secure networks, make sure you configure the provisioners on both profiles.
+
+
+Passthroughs required for Android
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Android devices require passthroughs to be created to allow them to fetch the configuration application from the Play Store.
+
+IMPORTANT: Passthroughs will vary depending on the location where your google account was created. You will need to add some extra passthroughs for the store of your country. In the section debug there is an how to determine which address you need to add.
+
+Add the following to the "Trapping" section of the Configuration tab in the PacketFence GUI.
+
+----
+passthrough=enabled
+passthroughs=*.ggpht.com,*.googleusercontent.com,android.clients.google.com,
+  *.googleapis.com,*.android.clients.google.com,*.gvt1.com
+----
+
+Step 3: Appendix
+~~~~~~~~~~~~~~~~
+
+Debugging MSPKI integration with PacketFence
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This a way to do the procedure of enrollment manually, mainly use for debugging purpose.
+
+First you need to generate a request and its private key via mkrequest. Go in the folder were sscep is installed, it should be /usr/local/scep
+
+ # cd /usr/local/scep
+
+And launch the command to generate the request
+
+ # ./mkrequest -email user@example.com
```

Comment:
That command will not work.
Because it requires modifying the mkrequest file that comes with sscep (maybe I should not package it).
Rewrite that to use the openssl command directly.